ydr/ui.py

```diff
@@ -144,14 +144,25 @@ class SOLLUMZ_PT_LIGHT_PANEL(bpy.types.Panel):
     bl_space_type = "PROPERTIES"
     bl_region_type = "WINDOW"
     bl_context = "data"
+    bl_options = {"DEFAULT_CLOSED"}
 
     @classmethod
     def poll(cls, context):
-        return context.light and context.active_object.sollum_type == SollumType.LIGHT
+        return context.light and context.active_object is not None
 
     def draw(self, context):
         layout = self.layout
         layout.use_property_split = True
+
+        aobj = context.active_object
+
+        if not aobj or aobj.sollum_type == SollumType.NONE or aobj.sollum_type != SollumType.LIGHT:
```

Comment:
This condition can be simplified to:
```suggestion
        if aobj.sollum_type != SollumType.LIGHT:
```
`aobj` shouldn't be `None` because it is already checked in `poll()` and the `== SollumType.NONE` check is redundant with the `!= SollumType.LIGHT` check.